src/MessageContainer.js

```diff
@@ -136,6 +136,7 @@ export default class MessageContainer extends React.PureComponent {
           ListFooterComponent={this.renderHeaderWrapper}
           ListHeaderComponent={this.renderFooter}
           {...this.props.listViewProps}
+          isLoadingEarlier={this.props.isLoadingEarlier}
```

Comment:
can you add isLoadingEarlier to propTypes to avoid  ci/circleci problem ??

Reply:
[FlatList](https://facebook.github.io/react-native/docs/flatlist#props) does not seem to have an `isLoadingEarlier` prop

Reply:
`FlatList` takes an `extraData` prop for forcing re-rendering. Check [this](https://facebook.github.io/react-native/docs/flatlist#extradata)

Reply:
Yes that would work i think.